spec/models/orders_spec.rb

```diff
@@ -46,8 +46,8 @@
     end
 
     it "can add an item" do
-      user = User.create!(first_name: "Testy", last_name: "McTest", password: "testing", email: "tester@testmail")
-      order = user.orders.create!(status: "ordered")
+      user = create(:user)
```

Comment:
Good use of Factory Bot for refactoring purposes. Tests render much more clearly.